gitops/build/main.yaml

```diff
@@ -79,7 +79,7 @@ stages:
               # Run
               hadolint ${DOCKERFILE_PATH}
             displayName: Dockerfile Lint
-            continueOnError: ${{ parameters.dockerLint.ignoreRuleViolations }}
+            continueOnError: and(and(succeeded(), ${{ parameters.dockerLint.ignoreRuleViolations }}), eq(variables['Build.sourceBranch'], '${{ parameters.sourceBranch }}'))
```

Comment:
This check has been problematic in the past. Isn't it possible to pass the condition in as a parameter?

Reply:
@ulrikstrid yep, this should be a condition and not continueOnError. I'll update.

Reply:
The problem is that `eq(variables['Build.sourceBranch'], '${{ parameters.sourceBranch }}')` has been changed from under us before which has broken pipelines. I would like to not have it hardcoded in the pipeline but as a par ameter that can be passed into the template.